docs/source/api/v5/deliveryservice_requests.rst

```diff
@@ -567,7 +485,7 @@ The response will be a representation of the created :term:`Delivery Service Req
 			"innerHeaderRewrite": null,
 			"ipv6RoutingEnabled": true,
 			"lastHeaderRewrite": null,
-			"lastUpdated": "2020-02-13T16:43:54Z",
+			"lastUpdated": "2023-06-09T10:51:00+05:30",
```

Comment:
Do we not need milliseconds?

Reply:
Again This is the Http Post Body Data. We cannot add milliseconds in this case. This data is part of requested {} data that is returned to http operations.